src/ps/commands/joinphrases.ts

```diff
@@ -0,0 +1,103 @@
+import { addJoinphrase, deleteJoinphrase, getJoinphrase } from '@/database/joinphrases';
+import { ChatError } from '@/utils/chatError';
+
+import type { NoTranslate, ToTranslate } from '@/i18n/types';
+import type { PSCommand } from '@/types/chat';
+
+export const command: PSCommand = {
+	name: 'jp',
+	help: 'Joinphrases module!',
+	perms: ['room', 'driver'],
+	syntax: 'CMD',
+	aliases: ['joinphrase'],
+	categories: ['utility'],
+	extendedAliases: {
+		addjp: ['jp', 'new'],
```

Comment:
Could you also add `ajp`, `djp`, and `ejp`? Those are currently the most commonly used ones